geozero/src/mvt/mod.rs

```diff
@@ -38,6 +38,8 @@ pub(crate) mod conversion {
     }
 }
 
+pub mod mvt_error;
```

Comment:
I wouldn't want to expose this as a namespace.

```suggestion
mod mvt_error;
pub use mvt_error::MvtError;
```

Reply:
I originally had what you're suggesting. There was some reason I changed it but I'll try and change it back!